pkg/controllers/management/auth/manager.go

```diff
@@ -436,6 +441,9 @@ func (m *manager) grantManagementPlanePrivileges(roleTemplateName string, resour
 		}
 	}
 
+	desiredBytes, _ := json.MarshalIndent(desiredRBs, "", "    ")
+	logrus.Debugf("desiredRBs: %v", string(desiredBytes))
```

Comment:
same

Reply:
Looks like I committed some other changes I made. These other files were not supposed to get added, going to remove them